dataprofiler/profilers/profile_builder.py

```diff
@@ -1531,21 +1546,24 @@ def save(self, filepath: str = None) -> None:
         :type filepath: String
         :return: None
         """
-        # Create dictionary for all metadata, options, and profile
```

Comment:
above in `save` we need to add the `type: str | None` with default `None` where we can choose '"pickle "| "json"`

Reply:
done